src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -300,69 +292,77 @@ const Embedding = (props) => {
   };
 
   return (
-    <div
-      className='vitessce-container vitessce-theme-light'
-      style={{ width, height, position: 'relative' }}
-      // make sure that the crosshairs don't break zooming in and out of the embedding
-      onWheel={() => { setCellInfoVisible(false); }}
-      onMouseMove={() => {
-        if (!cellInfoVisible) {
-          setCellInfoVisible(true);
-        }
-      }}
-      onMouseLeave={clearCellHighlight}
-      onClick={clearCellHighlight}
-      onKeyPress={clearCellHighlight}
-    >
-      {renderExpressionView()}
-      {
-        data ? (
-          <Scatterplot
-            cellOpacity={0.8}
-            cellRadius={cellRadius}
-            setCellHighlight={setCellHighlight}
-            theme='light'
-            uuid={embeddingType}
-            viewState={view}
-            updateViewInfo={updateViewInfo}
-            cells={convertedCellsData}
-            mapping='PCA'
-            setCellSelection={setCellsSelection}
-            cellColors={cellColorsForVitessce}
-            setViewState={setViewState}
-            getExpressionValue={getExpressionValue}
-            getCellIsSelected={getCellIsSelected}
-          />
-        ) : ''
-      }
-      {
-        createClusterPopover
-          ? (
-            <ClusterPopover
-              visible
-              popoverPosition={cellCoordinatesRef}
-              onCreate={onCreateCluster}
-              onCancel={() => setCreateClusterPopover(false)}
+    <>
+      {showLoader && <center><Loader experimentId={experimentId} size='large' /></center>}
```

Comment:
Rendering performance improvement. Render the `Scatterplot` once, then hide it when showing the `Loader` instead of removing it. This way we can just show it again when loading is finished